cmd/benchmark.go

```diff
@@ -46,12 +46,13 @@ type rawBenchmarkCmdArgs struct {
 	numOfFolders   uint
 
 	// options from flags
-	blockSizeMB float64
-	putMd5      bool
-	checkLength bool
-	blobType    string
-	output      string
-	mode        string
+	blockSizeMB   float64
+	putBlobSizeMB float64
```

Comment:
This variable has multiple versions in below code from float64, float32 to int64. Can we check if existing variable of blockSizeMB shall also be uniform across the code and covert this one as well.
Also, why do we take it as float64, Is user allowed to give something like 0.5 here?

Reply:
Yes, a user can give 0.5, etc to indicate less than 1 MB. Any user facing input is taking in float64, the float32 we see below is just an artifact of how we test, so I wouldn't worry about it too much since we are replacing that anyway

Reply:
there are other instances taking it in int format also, if user sent a float number do we convert it to number of bytes and then store in int or there will be potential loss of data while converting to int.

Reply:
So what we do is transform the param from float (parameter is in MB) to int (these parameters are in bytes since we multiply by MB)